zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/CallActivityTest.java

```diff
@@ -766,11 +763,9 @@ public void shouldPropagateTreePathPropertiesWithIncident() {
     final String rootProcessId = "root";
     final String callActivity1Id = "callParent";
     final String callActivity2Id = "callChild";
-    final HashFunction hashFunction = Hashing.murmur3_128();
-    final var ca1Hash =
-        hashFunction.hashString(callActivity1Id, Charset.defaultCharset()).toString();
-    final var ca2Hash =
-        hashFunction.hashString(callActivity2Id, Charset.defaultCharset()).toString();
+    // every deployment resource will have indexes starting from zero
+    final var ca1Index = 0;
+    final var ca2Index = 0;
```

Comment:
🔧 These definitions were unclear to me, until I read the test another time. I suggest moving them closer to the assertion (the only place where these are used). You could also consider combining them into a more descriptive variable, e.g. `FIRST_CALL_ACTIVITY_INDEX = 0`.

~🔧 This highlights that it seems that we don't have any tests verifying the correct behavior of the lexicographic index as part of the calling element path, e.g. where the call activity is not the first one appearing in the process~ EDIT: resolved by the next commit.